src/entity/entity.js

```diff
@@ -57,6 +57,28 @@ class Entity {
     pause() {
 
     }
+
```

Comment:
I had this in Player originally, but I moved it up to entity since it might be cool to have enemies turn to face the player too (especially if they're turrets or something).

Reply:
Cool! We might be able to refactor this into a utils class as well if we end up adding a lot. Let's keep it in class for now to reduce scope.